_gsocproposals/2019/proposal_ROOTzstd.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Novel Applications of Zstandard (ZSTD) compression algorithm to ROOT
+layout: gsoc_proposal
+project: ROOT
+year: 2019
+organization:
+  - UNL
+---
+
+## Description
+ROOT [1] is a framework for data processing, born at CERN, at the heart of the research on high-energy physics. ROOT consists of multiple components such as I/O, a C++ interpreter, GUI, and math libraries.  One of its tasks is to reduce the on-disk usage of physics data; a typical approach is to use various compression techniques.  ROOT I/O subsystem supports data compression using common compression algorithms such as ZLIB, LZMA and LZ4.
```

Comment:
In case you want to have a space in between both paragraphs, you need add another new line at the end.